### ISSUE TITLE
Personal notes / personal rating system

### ISSUE THREAD

**Issue by 4lv-2** [mihonapp/mihon#643]
### Describe your suggested feature

Writing notes and giving ratings that only show for one's library. Even a few words would be great.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
You mean like what the PR literally called "Add Manga Notes" (#428) is trying to do? Please go and check last issues and PRs (open AND closed) before creating a new issue.

**Comment by 4lv-2** [comment]
Oh, sorry checked alot and didn't see that...
But I don't have the feature 
Is it on the preview version..?

**Comment by MajorTanya** [comment]
> Oh, sorry checked alot and didn't see that...
> But I don't have the feature 
> Is it on the preview version..?

No, it isn't in the code at this point, hence the PR still being open.

**Comment by Xori71** [comment]
> Oh, sorry checked alot and didn't see that... But I don't have the feature Is it on the preview version..?

Well, that’s why they are called Pull _Requests_. The people responsible for the changes/additions want to add them to the main branch, but that is to the discretion of the owner.
